obal/data/modules/srpm.py

```diff
@@ -0,0 +1,89 @@
+"""
+Build SRPM
+"""
+
+import shutil
+import os
+import commands
```

Comment:
This module is deprecated since Python 2.6 and removed in Python 3 according to https://docs.python.org/2.7/library/commands.html.